packages/styles/scss/patterns/blocks/content-block-simple/_content-block-simple.scss

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '../../../globals/imports';
+@import '../../sub-patterns/card/index';
+@import '../../../components/link-with-icon/link-with-icon';
+
```

Comment:
> The image needs a `max-width` of `640px` or `80mu`.

Should this not be taken care of in the `Image` component?

Reply:
Not from what I can see at the moment. Each pattern handles Images slightly different. For instance it would be weird for a card or leadspace to have a max-width.

Reply:
Ok sorry, I don't think i know what you mean then?

Reply:
Spacing around the image is also incorrect. `48px` on top and bottom except on small where it's `32px` top and bottom.